<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderDetailsWidget</class>
 <widget class="QWidget" name="OrderDetailsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Order Details</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="numberHorizontalLayout">
        <item>
         <widget class="QLabel" name="orderNumberLabel">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Order #xx</string>
          </property>
          <property name="textFormat">
           <enum>Qt::RichText</enum>
          </property>
          <property name="openExternalLinks">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="collapseButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../res.qrc">
            <normaloff>:/res/icons/collapse_right.png</normaloff>:/res/icons/collapse_right.png</iconset>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="orderStatusLabel">
        <property name="text">
         <string>Status: x</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="datesHorizontalLayout">
        <item>
         <widget class="QLabel" name="createdAtLabel">
          <property name="text">
           <string>Created: 0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="updatedAtLabel">
          <property name="text">
           <string>Updated: 0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="CollapsibleWidgetHeader" name="addressHeader" native="true">
     <property name="title" stdset="0">
      <string>Address:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="addressContainer" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="nameHorizontalLayout">
        <item>
         <widget class="ClickyLineEdit" name="addressNameEdit">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="placeholderText" stdset="0">
           <string>Name</string>
          </property>
          <property name="type" stdset="0">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ClickyLineEdit" name="addressOrgEdit">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="placeholderText" stdset="0">
           <string>Organization</string>
          </property>
          <property name="type" stdset="0">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="ClickyLineEdit" name="address1Edit">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="placeholderText" stdset="0">
         <string>Address 1</string>
        </property>
        <property name="type" stdset="0">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ClickyLineEdit" name="address2Edit">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="placeholderText" stdset="0">
         <string>Address 2</string>
        </property>
        <property name="type" stdset="0">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ClickyLineEdit" name="addressCityEdit">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="placeholderText" stdset="0">
         <string>City, State</string>
        </property>
        <property name="type" stdset="0">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="zipHorizontalLayout">
        <item>
         <widget class="ClickyLineEdit" name="addressCountryEdit">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="placeholderText" stdset="0">
           <string>Country</string>
          </property>
          <property name="type" stdset="0">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ClickyLineEdit" name="addressZipEdit">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="placeholderText" stdset="0">
           <string>Postal Code</string>
          </property>
          <property name="type" stdset="0">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="emailHorizontalLayout">
        <item>
         <widget class="ClickyLineEdit" name="addressPhoneEdit">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="placeholderText" stdset="0">
           <string>Phone #</string>
          </property>
          <property name="type" stdset="0">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ClickyLineEdit" name="addressEmailEdit">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="placeholderText" stdset="0">
           <string>Email</string>
          </property>
          <property name="type" stdset="0">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="copyAllHorizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="addressCopyAllButton">
          <property name="text">
           <string>Copy all</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="CollapsibleWidgetHeader" name="itemsHeader" native="true">
     <property name="title" stdset="0">
      <string>Items:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="itemsContainer" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTreeWidget" name="itemsTreeWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="indentation">
         <number>10</number>
        </property>
        <column>
         <property name="text">
          <string>Product</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Unit Price</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Price</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="itemsTotalLabel">
        <property name="font">
         <font/>
        </property>
        <property name="text">
         <string>Totals</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="CollapsibleWidgetHeader" name="shippingHeader" native="true">
     <property name="title" stdset="0">
      <string>Shipping:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="shippingContainer" native="true">
     <layout class="QFormLayout" name="formLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="shippingWeightLabel">
        <property name="text">
         <string>Weight:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="shippingWeightValueLabel">
        <property name="text">
         <string>0 gr</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="shippingMethodLabel">
        <property name="text">
         <string>Method:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="shippingMethodValueLabel">
        <property name="text">
         <string>Method</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="shippingTrackingLabel">
        <property name="text">
         <string>Tracking:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="shippingTrackingRequiredLabel">
        <property name="text">
         <string>Not required</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="shippingTrackingNoLabel">
        <property name="text">
         <string>Tracking #:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="ClickyLineEdit" name="shippingTrackingNoEdit">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="placeholderText" stdset="0">
         <string>Untracked</string>
        </property>
        <property name="type" stdset="0">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="shippingTrackingUrlLabel">
        <property name="text">
         <string>Tracking URL:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="ClickyLineEdit" name="shippingTrackingUrlEdit">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="placeholderText" stdset="0">
         <string>Untracked</string>
        </property>
        <property name="type" stdset="0">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="shippingSubmitButton">
        <property name="text">
         <string>Mark Shipped</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="shippingPackagingLabel">
        <property name="text">
         <string>Packaging:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="shippingPackagingValueLabel">
        <property name="text">
         <string>Not packaged</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="CollapsibleWidgetHeader" name="billingHeader" native="true">
     <property name="title" stdset="0">
      <string>Billing:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="billingContainer" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <widget class="QPushButton" name="customerInvoiceButton">
          <property name="text">
           <string>Customer Invoice</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sellerInvoiceButton">
          <property name="text">
           <string>Seller Invoice</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="billingLabel">
          <property name="font">
           <font/>
          </property>
          <property name="text">
           <string>Billing</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="billingLinkLabel">
          <property name="text">
           <string>Payment</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="openExternalLinks">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>12</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonHorizontalLayout">
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QPushButton" name="prevOrderButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="icon">
        <iconset resource="../res.qrc">
         <normaloff>:/res/icons/arrow_left.png</normaloff>:/res/icons/arrow_left.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="orderPosLabel">
       <property name="text">
        <string>0/0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextOrderButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="icon">
        <iconset resource="../res.qrc">
         <normaloff>:/res/icons/arrow_right.png</normaloff>:/res/icons/arrow_right.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickyLineEdit</class>
   <extends>QFrame</extends>
   <header>widgets/clickylineedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CollapsibleWidgetHeader</class>
   <extends>QWidget</extends>
   <header>widgets/collapsiblewidgetheader.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
